src/kirby/package.json

```diff
@@ -24,7 +24,7 @@
   "homepage": "https://github.com/kirbydesign/designsystem#readme",
   "types": "index.d.ts",
   "dependencies": {
-    "@ionic/angular": "^4.1.1",
+    "@ionic/angular": "^4.6.0",
```

Comment:
Should this not be a specific version too?

Reply:
Should be fixed now :)